.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,24 @@
+name: Shellcheck on scripts
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up system
+      shell: bash
+      run: |
+        sudo apt-get update -qq
+        sudo apt-get install shellcheck
+    - uses: actions/checkout@v1
+    - name: Run shellcheck
+      run: |
+        shellcheck \
+          tools/bisect-git-annex.scripts/bisect-git-annex-lock.sh \
+          tools/ci/install-annex.sh \
+          tools/ci/install-minimum-git.sh \
+          tools/ci/prep-travis-devel-annex.sh \
+          tools/ci/prep-travis-forssh.sh
```

Comment:
> which ATM are shellcheck kosher

I think it's worth adding a code comment explaining that you're only testing the subset of scripts that pass shellcheck.

I was curious about what this looked like overall:

```console
$ git grep -l '^#!/.*sh$' tools/ | wc
     29      29     851
$ shellcheck $(git grep -l '^#!/.*sh$' tools/) | grep -c "In tools/"
144
```

Reply:
added a comment, thanks.  Some scripts just need minor fixups to become shellcheck compliant, but I did not embark on that mission